core/src/main/kotlin/fr/sncf/osrd/api/pathfinding/PathfindingResultConverter.kt

```diff
@@ -124,7 +124,7 @@ private fun makePendingOPWaypoint(
 private fun makePendingUserDefinedWaypoint(
     infra: RawSignalingInfra,
     path: PathProperties,
-    pathOffset: Offset<Path>
+    pathOffset: Offset<BlockPath>
```

Comment:
This must match `makePendingOPWaypoint`:
```suggestion
    pathOffset: Offset<TravelledPath>
```
And it propagates correctly to `PathProperties.getTrackLocationAtOffset` if I'm correct.